### ISSUE TITLE
sty is coming out stretched and squashed 

### ISSUE THREAD

**Issue by MikeDmaker** [Ultimaker/Cura#19634]
### Cura Version

5.8.0

### Operating System

macOS

### Printer

Anycubic kobra max 

### Reproduction steps

I add the sty file, scale it uniformly to the desired size, it slices very slowly then I preview and it all looks fine 

### Actual results

the file in shorter on the z axis and stretched on the x or y (varies) tried a few times and it comes out different sizes each time 

### Expected results

normal print 

### Add your .zip and screenshots here ⬇️

as above

**Comment by GregValiant** [comment]
Thanks for the report.
Do you mean STL file?
This is one I've never heard of and I can't duplicate it on my own.  Without a project file there is nothing that can be done.

**Comment by MikeDmaker** [comment]
yeah sorry stl file, oddly its happening with more than one file, or I would have attached one to test

**Comment by MikeDmaker** [comment]
Hi sorry yeah please close it On 12 Sep 2024, at 12:40, GregValiant ***@***.***> wrote:﻿
Without a project file we have nothing to work with.  Should I close this?
